german/Stopfen/stopfen.md

```diff
@@ -12,3 +12,5 @@ Was machen Sie nun?
 [Aus dem Traum aufwachen und Oktoberfest besuchen](../Oktoberfest/oktoberfest.md)
 
 [Die Stille geniessen und nachdenken](entspannen/entspannen.md)
+
+[Eine flasche Tequila trinken](Alkohol/alkohol.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.